sphinx_rtd_theme/__init__.py

```diff
@@ -5,14 +5,13 @@
 """
 
 from os import path
+import pkg_resources
 
 import sphinx
 
-
-__version__ = '0.4.3.dev0'
+__version__ = pkg_resources.get_distribution('sphinx_rtd_theme').version
```

Comment:
This is the way I saw it most online, but if there is a better way I'm all for changing it.

The reason for putting the version in a file is to be able to inspect the version just by looking at the directory contents. I don't know if it's idiomatic Python and I can imagine this might not work for the rtd.org repo. Is there a way to test if `file:` works there?

Reply:
@agjohnson I just tested an install into `docs/_themes` and you are right that this will report the wrong version. If we want to support installations from master, we can not use this method. We can still use it however if we drop support for installing from master and do more frequent dev releases.

On the issue of `file:` in `setup.cfg`, it's probably safer to just hard code it (which should work on rtd.org). We can always move it to a file if we need to later.

Reply:
Aye, perhaps hard coding in setup.cfg is the best pattern. We're using it already on a few packages. I'd check the readthedocs/readthedocs.org repo for how we're loading the version. I believe that repo has the common code we use for loading the setup.cfg into the application to get the version.